packages/ckeditor5-list/src/documentlistproperties/documentlistpropertiesediting.js

```diff
@@ -77,6 +72,9 @@ export default class DocumentListPropertiesEditing extends Plugin {
 			model.schema.extend( '$container', { allowAttributes: strategy.attributeName } );
 			model.schema.extend( '$block', { allowAttributes: strategy.attributeName } );
 			model.schema.extend( '$blockObject', { allowAttributes: strategy.attributeName } );
+
+			// Register downcast strategy.
+			documentListEditing.registerDowncastStrategy( strategy );
```

Comment:
I'd explicitly pass those properties from the list prop strategy that are necessary for downcast.

Reply:
:+1: